### PR TITLE
feat: Question Number at the top

### LINKED ISSUE #94: Enchancement: Question number at the top
I was wondering if the question number can go up into the h1 heading as some of the questions are a little longer and it stretches across the entire screen; while the actual question goes into a paragraph tag below it.

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #94

Current Look 

I am going to be playing the size values sooner than later. I just wanted to get it started.

### CODE REVIEW

**Comment by jdwilkin4** [comment]
> I am going to be playing the size values sooner than later. I just wanted to get it started.

Does this mean that this PR is still a work in progress? 
If so, you can amend your PR title to include WIP in the beginning. 

Otherwise, I can go ahead of review it.

Just let me know. 

Thanks 👍

**Comment by a2937** [comment]
You can review it. I am in need of guidance on the font size for the question.

File: src/components/Questions.tsx [hunk 1]
```diff
@@ -25,7 +25,11 @@ interface QuizProps {
 const Questions: React.FC<QuizProps> = (QuizProps) => {
   return (
     <>
-      <h1 className="quiz-heading">{QuizProps.currQuestion.Question}</h1>
+      <h1 className="quiz-heading">Question {QuizProps.questionNumber}</h1>
+      <br />
+      <div className="quiz-text">
+        <p>{QuizProps.currQuestion.Question}</p>
```
Comment (jdwilkin4) [inline src/components/Questions.tsx:31]: I think using the paragraph element is to small. 

I think it would be better to use an `h2`

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Pull request merged**